metecho/api/views.py

```diff
@@ -198,6 +212,45 @@ class UserViewSet(mixins.RetrieveModelMixin, mixins.ListModelMixin, GenericViewS
     queryset = User.objects.all()
 
 
+class ProjectDependencyViewSet(ReadOnlyModelViewSet):
+    """Dependencies available during Project creation"""
+
+    permission_classes = (IsAuthenticated,)
+    serializer_class = ProjectDependencySerializer
+    pagination_class = CustomPaginator
+    queryset = ProjectDependency.objects.all()
+
+
+class GitHubOrganizationViewSet(ReadOnlyModelViewSet):
+    """GitHub Organizations available during Project creation"""
+
+    permission_classes = (IsAuthenticated,)
+    serializer_class = GitHubOrganizationSerializer
+    pagination_class = CustomPaginator
+    queryset = GitHubOrganization.objects.all()
+
+    @extend_schema(request=None, responses={202: None})
+    @action(detail=True, methods=["POST"])
+    def members(self, request, pk):
+        """Queue a job to fetch the members of the Organization"""
+        org: GitHubOrganization = self.get_object()
+        org.queue_get_members(user=request.user)
+        return Response(status=status.HTTP_202_ACCEPTED)
+
+    @extend_schema(request=CheckRepoNameSerializer, responses={202: None})
+    @action(detail=True, methods=["POST"])
+    def check_repo_name(self, request, pk):
+        """Queue a job to determine if a repository name is available for the Organization"""
+        org: GitHubOrganization = self.get_object()
+        serializer = CheckRepoNameSerializer(data=request.data)
+        serializer.is_valid(raise_exception=True)
+        org.queue_check_repo_name(
+            name=serializer.validated_data["name"],
+            originating_user_id=str(request.user.id),
+        )
+        return Response(status=status.HTTP_202_ACCEPTED)
```

Comment:
@jerivas I notice that the `feature_branches` action does a similar operation here, but in the request-response cycle instead of in a queued job. What do you think about doing that for `check_repo_name` as well? It would be somewhat cleaner on the front end, since I'd rather not store this in the Redux store -- and our WS setup goes through the store.

I'm okay leaving the `members` action asynchronous, since I see some value in caching those results in the Redux store anyway.

Reply:
Oh, I didn't know about this endpoint. I thought all interactions with the GH API were off-banded, but as you said, this is easier. I'm ok converting both `check_repo_name` and `members` to this approach 👍 I'll do that now

Reply:
Fixed in 26322eedcf9a0637fd988d87578603d3772dbf45